packages/js/data/src/index.ts

```diff
@@ -6,7 +6,6 @@ import '@wordpress/core-data';
 // Export store names
 export { SETTINGS_STORE_NAME } from './settings';
 export { PLUGINS_STORE_NAME } from './plugins';
-export { ONBOARDING_STORE_NAME } from './onboarding';
```

Comment:
Hmmm should we remove this export in one go? It could break plugins using it, but I am not entirely sure how we can deprecate a `const` in a good manner.

Having said that, I don't think it hurts still exporting this as using `select( store_string_name )` is still expected to work. I think just recommending to use `onboardingStore` in our docs would suffice.

Reply:
That's a great point I didn't consider 3rd party plugins using it! I'll put it back